src/fides/core/pull.py

```diff
@@ -66,6 +68,40 @@ def pull_existing_resources(
     return existing_keys
 
 
+def pull_resource_by_key(
+    manifests_dir: str,
+    url: str,
+    headers: Dict[str, str],
+    fides_key: str,
+    resource_type: str,
+) -> None:
+    """
+    Pull a resource from the server by its fides_key and update the local manifest file.
+    """
+    if manifests_dir[-1] == "/":
+        manifests_dir = manifests_dir[:-1]
+    manifest_path = f"{manifests_dir}/{resource_type}.yaml"
+    print(f"Pulling resource with fides_key: {fides_key}...", end=" ")
```

Comment:
Maybe add the resource type to this error message too  and add error handling if the resource_type is unexpected